### PR TITLE
improvements on the first-sec div at the about page

### PR DESCRIPTION
**Pull request by holo-dango**
Put the first section bit inside a wrapper that facilitates a grid layout. Includes changes to styles.css

### CODE REVIEW

**Comment by GoldElysium** [comment]
Lemme know if this is done, then I'll check it for you

File: styles.css [hunk 1]
```diff
@@ -57,12 +57,42 @@ body {
...
+  padding-right: 100px;
   text-align: left;
   background: #FFFFFF;
   font-size: 30px;
-  margin: 200px 300px 200px 300px;
+  margin-left: 300px;
+  margin-right: 300px;
+  margin-top: 200px;
+  margin-bottom: 200px;
+} */
```
Comment (GoldElysium) [inline styles.css:71]: Make sure to remove this later on if this code is not needed anymore

File: hololive-about.html [hunk 2]
```diff
@@ -30,12 +30,14 @@
...
   	</div>
 
-    <!--First Section of Body-->
-
-    <div class="first-sect">
-      <h3>What is Hololive??</h3>
-      <p>Test Sentence</p>
-    </div>
+		<article class="about-grid-container">
+    	<!--First Section of Body-->
```
Comment (GoldElysium) [inline hololive-about.html]: Not a big issue, but indent this correctly

**Review by GoldElysium — APPROVED** [review]

**Comment by holo-dango** [comment]
Opted to create a new pull request using a more recent version of the master branch with indentation fixed and unused commented styles removed.
https://github.com/HL-Resistance/hlresistance-preview.github.io/pull/11
Changes are the same, scope being the about page and styles.css

**Comment by GoldElysium** [comment]
Branch removal in favor of #11

**Pull request closed without merging**